roles/openshift_node/tasks/systemd_units.yml

```diff
@@ -9,19 +9,14 @@
   template:
     dest: "/etc/systemd/system/{{ openshift_service_type }}-node.service"
     src: "node.service.j2"
-  when: not l_is_node_system_container | bool
+  when: not openshift_is_atomic | bool
   notify:
   - reload systemd units
 
-- block:
-  - name: Remove existing systemd service
```

Comment:
This task breaks idempotency of upgrades.  Currently, atomic CI upgrade check appears to not actually be upgrading a cluster from 3.9 to 3.10 and this is causing the systemd unit to not be placed via the atomic cli.

Reply:
You mean `fedora/27/atomic/update`? It does a minor upgrade from 3.10 to another 3.10

I'm working on 3.9 -> 3.10 atomic upgrade in https://github.com/openshift/openshift-ansible/pull/8273, I'll rename tasks there